lib/model/folder.go

```diff
@@ -997,6 +1006,10 @@ func (f *folder) monitorWatch(ctx context.Context) {
 			}
 			lastWatch = time.Now()
 			watchaggregator.Aggregate(aggrCtx, eventChan, f.watchChan, f.FolderConfiguration, f.model.cfg, f.evLogger)
+			if fs.WatchKqueue {
```

Comment:
On mobile so asking instead of investigating; is there a risk of ending up here more than once, so that there should be a == null check to avoid leaking subscriptions?

Reply:
I am pretty sure it can't happen (we only get here after a failure/error from the watch, where we do unsubscribe), however there's no point un/subscribing on failure for this (warn once if watching is enabled) -> removed.